### ISSUE TITLE
Update angular dependency

### ISSUE THREAD

**Issue by jecraig** [KingSora/OverlayScrollbars#210]
Cannot use the Angular cli update command to update to Angular 9 without --force because the packages.json marks 7 as the latest supported version.

```
C:\Code\d8\web\aeclient>ng update @angular/core
Using package manager: 'npm'
Collecting installed dependencies...
Found 76 dependencies.
Fetching dependency metadata from registry...
                  Package "overlayscrollbars-ngx" has an incompatible peer dependency to "@angular/core" (requires "^7.0.0" (extended), would install "9.1.3").
× Migration failed: Incompatible peer dependencies found.
Peer dependency warnings when installing dependencies means that those dependencies might not work correctly together.
You can use the '--force' option to ignore incompatible peer dependencies and instead address these warnings later.
```

**Comment by KingSora** [comment]
Good day!

Thanks, I'll fix this as soon as possible.

**Comment by TokkeDev** [comment]
What's the status on this? 😊

**Comment by KingSora** [comment]
@TokkeDev I'm sorry I lost the sight of this issue because I focused so hard on `v2.0.0`. I'll do my best to solve this in the next release.

**Comment by TokkeDev** [comment]
No worries 😊 That sounds good 👌

**Comment by lovelyc4t** [comment]
Hi @KingSora first of all, really love the plugin, kudos! ❤️ Is there any chance of bumping angular dependencies soon? Best love!

**Comment by TokkeDev** [comment]
@KingSora, everything seems to be working just fine with both Angular 9, Angular 10 and Angular 11, so shouldn't we just bump the peer dep version af Angular to accept 11? 😊

**Comment by KingSora** [comment]
@TokkeDev @lovelyc4t @jecraig 
Fixed with [version 1.13.1](https://github.com/KingSora/OverlayScrollbars/releases/tag/1.13.1) / `"overlayscrollbars-ngx": "0.2.3"`
